community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,335 @@
+.. _doc_pr_review_guidelines:
+
+Pull request review guidelines
+==============================
+
+.. note::
+	This page is targeted at engine maintainers responsible for reviewing
+	and approving pull requests. While not all tips and recommendations
+	here are actionable if you are not a maintainer, this can still give
+	you and insight into what goes into successfully merging a PR.
+
+	Even if you are not a maintainer, you can always help by spotting
+	issues in code or problems with the implementation overall, as well
+	as by doing live testing of PRs on your machine and confirming that
+	they work as intended.
+
+If you are a designated Godot contributor and maintainers, you've likely
+demonstrated skill, knowledge and capacity for better judgement when
+writing, reviewing and improving Godot engine code and experience. You
+are entrusted with keeping Godot moving forward, so feel free to exercise
+your maintainer power to achieve that.
+
+One part of that power is reviewing and merging pull requests, created
+by your teammates and other contributors. You may not even realize it,
+but you have full control of that green "Merge" button and are encouraged
+to use it. But there are a few rules, checks and recommendations that you
```

Comment:
Honestly I think we should put more emphasis on reviewing rather than merging. In an ideal world many reviews would come before a merge.

Reply:
We basically have two problems here, and two articles combined into one. Both are important to address, and this is just the first draft. I can try to shift the emphasis, but I think this statement also sends the message that we wanted to send about having that power and being able to exercise it.

Reply:
I agree with @pycbouh, ideally we would have a "how to review PRs" doc and a "how to merge PRs" doc.